src/utils/css_styler.tsx

```diff
@@ -1,11 +1,41 @@
-// Utility functions to add CSS to HTML style element
+// Utility functions to concatenate view components CSS
+import {IDictionary} from '../models';
 
-let pageCSS = '';
+// Concatenate each component CSS into this variable
+let componentsCSS = '';
 
-export function addCSSToHead(css: string): void {
-  pageCSS += css;
+// Media queries width in px configuration
+const MEDIA_QUERIES_WIDTH_CONFIG: IDictionary<string> = {
+  xsMin : '320px', // Custom, iPhone Retina
+  smMin : '480px', // Extra Small Devices, Phones
+  mdMin : '768px', // Small Devices, Tablets
+  lgMin : '992px', // Medium Devices, Desktops
+  xlgMin: '1200px' // Large Devices, Wide Screens
+};
+
+// Supported media queries options
+interface IMediaQueries {
```

Comment:
@phidias This keeps breaking with the same error. From what I can tell it is not related to Webpack compilation or anything. Can you give it a try in your PC?